ext/build_defaults.yaml

```diff
@@ -24,6 +24,9 @@ build_msi:
   hiera:
```

Comment:
I can't comment on the line I want to comment on, but the puppet_for_the_win ref needs to be updated for this to not fail